tests/viz_tests.py

```diff
@@ -1330,3 +1331,43 @@ def test_get_aggfunc_non_numeric(self):
             viz.PivotTableViz.get_aggfunc("strcol", self.df, {"pandas_aggfunc": "min"})
             == "min"
         )
+
+
+class TestDistributionPieViz(SupersetTestCase):
+    base_df = pd.DataFrame(
+        data={
+            "intcol": [1, 2, 3, 4],
+            "floatcol": [0.1, 0.2, 0.3, 0.4],
```

Comment:
Would it help to insert a couple of Nones in these columns as well? And maybe add a boolean column, too?

Reply:
Good catch, numeric `null`s were not properly handled (showed up as `nan`), adding tests.